permissions/plugin-portshift-scanner.yml

```diff
@@ -1,9 +1,9 @@
 ---
-name: "portshift-scanner"
-github: "jenkinsci/portshift-scanner-plugin"
+name: "securecn-scanner"
+github: "jenkinsci/securecn-scanner-plugin"
 paths:
-  - "io/jenkins/plugins/portshift-scanner"
+  - "io/jenkins/plugins/securecn-scanner"
```

Comment:
```suggestion
  - "io/jenkins/plugins/portshift-scanner"
```
You can't rename the artifact id, because there's no way to upgrade from that.

Reply:
What can we do? if we want to rebrand the plugin?

Reply:
You can rebrand the displayed name and description.